index.html

```diff
@@ -70,5 +70,198 @@ <h2 class="first-title">I'm Peter <br/> Glad to see you! </h2>
 </div>
 </div>
 </section>
+<section class="card-works" id="portfolio">
+    <div class="card card-works-layout1">
+        <a href ="#">
+            <img src="/Images/SnapshootPortfolio1.png" alt="SnapshootPortfolio1" class="snapshot1">
+        </a>
+        <div class="card-works-content1">
+            <h3 class="tonic"> Tonic </h3>
```

Comment:
- [ ] Please use the proper padding and margin to the cards in your project. There should be a margin to the cards horizontally.